test/modules/core/lib/attribute/attribute-transition-manager.spec.ts

```diff
@@ -56,173 +51,150 @@ test('AttributeTransitionManager#constructor', t => {
   t.end();
 });
 
-if (device.info.type === 'webgl2') {
```

Comment:
This is hard to review due to the huge indentation-driven delta. If you are just removing the if statements then all is fine. Otherwise it might have been worth first landing a PR that just updated the indentation (e.g. by early exit if not webgl2)..

Reply:
It was indeed just a removal of the `if` statement